evalml/tests/automl_tests/test_automl.py

```diff
@@ -2430,3 +2430,20 @@ def test_automl_raises_deprecated_random_state_warning(X_y_multi):
         automl = AutoMLSearch(X_train=X, y_train=y, problem_type='multiclass', random_state=10)
         assert automl.random_seed == 10
         assert str(warn[0].message).startswith("Argument 'random_state' has been deprecated in favor of 'random_seed'")
+
+
+def test_automl_check_for_high_variance(X_y_binary, dummy_binary_pipeline_class, caplog):
```

Comment:
Not entirely in the scope of this issue but I just noticed we don't test if the high variance warning runs during search. I think it would be worth doing that eventually but we can tackle that in a separate issue. What do you think?

Reply:
Yup, you're right--we test that the output is bool but not whether we actually log. I'll add a test 🥳